deep-security-agent/tasks/install-linux.yml

```diff
@@ -0,0 +1,149 @@
+# Copyright 2018, Trend Micro
+#
+# License as per [repo](master/LICENSE)
+#
+# *********************************************************************
+# * Install the Deep Security Agent (For Linux only)
+# *********************************************************************
+---
+# *********************************************************************
+# * Detect the platform architecture and package manager
+# *********************************************************************
+- name: Setting facts (apt) 
+  set_fact:
+    agent_local_path: /tmp/ds_agent.deb
+  when: ansible_pkg_mgr == "apt"
+
+- name: Setting facts (yum, zypper) 
+  set_fact:
+    agent_local_path: /tmp/ds_agent.rpm
+  when: >
+        ansible_pkg_mgr == "yum" or
+        ansible_pkg_mgr == "zypper"
+
+- name: Setting facts (svr4pkg) 
```

Comment:
Can you comment with name or make a debug msg to tell which platform does this kine of package manager support?

Reply:
Agree, if it is no longer support we can remove it directly.

Reply:
svr4pkg is used for Solaris 5.10, and pkg5 is used for Solaris 5.11. Because we don't official support our playbook for Solaris, I will remove it directly.